docker/supabase/docker-compose.yml

```diff
@@ -9,7 +9,7 @@ services:
     environment:
       SUPABASE_URL: http://kong:8000
       STUDIO_PG_META_URL: http://meta:8080
-      SUPABASE_REST_URL: http://kong:8000/rest/v1/
+      SUPABASE_REST_URL: ${SUPABASE_URL}/rest/v1/
```

Comment:
Won't line 10 cause problems, since you're changing to the internal vs. public API url?

What if you literally use:

```
SUPABASE_REST_URL: https://dev.api.telescope.cdot.systems/v1/supabase/rest/v1
```

OR, should we alter `SUPABASE_URL` to always use the public api?  That is, forward `SUPABASE_URL` in line 10 vs. changing it?  The studio container is just a next.js web app, so it should be able to always use the public url, no?

Reply:
Let's not change line 10, since in supabase docker-compose they have it set to https://github.com/supabase/supabase/blob/618fd88a076ae4258b5d080f0d493ad562b837ee/docker/docker-compose.yml#L20
But would `SUPABASE_URL` in our docker-compose overwrite the `SUPABASE_URL` in our .env file?

Reply:
OK, makes sense re: line 10 and matching what they do.

I'm not sure if it will override or use your value.  Let's test your PR and see.